src/util/ImageUtil.js

```diff
@@ -0,0 +1,13 @@
+L.ImageUtil = {
+
+  getCmPerPixel: function(overlay) {
+    var map = overlay._map;
+
+    var dist = map
+      .latLngToLayerPoint(overlay.getCorner(0))
+      .distanceTo(map.latLngToLayerPoint(overlay.getCorner(1)));
+
+    return (dist * 100) / overlay._image.width;
```

Comment:
I was thinking about making the same thing today to draw polylines over the matching points for matcher, thank you for implementing this, very cool!